tests/JustSaying.UnitTests/Fluent/WhenUsingDefaultServiceResolver.cs

```diff
@@ -0,0 +1,62 @@
+using JustSaying.AwsTools;
+using JustSaying.Fluent;
+using JustSaying.Messaging.MessageSerialization;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Shouldly;
+using Xunit;
+
+namespace JustSaying.UnitTests.Fluent
+{
+    public class WhenUsingDefaultServiceResolver
+    {
+        private readonly DefaultServiceResolver _sut;
+
+        public WhenUsingDefaultServiceResolver()
+        {
+            _sut = new DefaultServiceResolver();
+        }
+
+        [Fact]
+        public void ShouldResolveILoggerFactory()
+        {
+            _sut.ResolveService<ILoggerFactory>().ShouldBeOfType<NullLoggerFactory>();
+        }
+
+        [Fact]
+        public void ShouldResolveIAwsClientFactoryProxy()
+        {
+            _sut.ResolveService<IAwsClientFactoryProxy>().ShouldBeOfType<AwsClientFactoryProxy>();
+        }
+
+        [Fact]
+        public void ShouldResolveIHandlerResolverAsNull()
+        {
+            _sut.ResolveService<IHandlerResolver>().ShouldBeNull();
+        }
+
+        [Fact]
+        public void ShouldResolveIMessagingConfig()
+        {
+            _sut.ResolveService<IMessagingConfig>().ShouldBeOfType<MessagingConfig>();
+        }
+
+        [Fact]
+        public void ShouldResolveIMessageSerializationFactory()
+        {
+            _sut.ResolveService<IMessageSerializationFactory>().ShouldBeOfType<NewtonsoftSerializationFactory>();
```

Comment:
Pedantic: either the assert should be "not null", or the test name should say what the default type gets resolved

Reply:
You are so damn picky Lang!

Reply:
Updated @slang25 👍

Reply:
Guilty 😄